src/utils/gfycatSdk.js

```diff
@@ -3,7 +3,11 @@
 
 import Gfycat from './gfycat-sdk.umd.bundle.min';
 
-export default new Gfycat({
-    client_id: '2_KtH_W5',
-    client_secret: '3wLVZPiswc3DnaiaFoLkDvB4X0IV6CpMkj4tf2inJRsBY6-FnkT08zGmppWFgeof',
-});
+let instance = null;
+
+export default function (key, secret) {
+    if (!instance) {
+        instance = new Gfycat({client_id: key, client_secret: secret});
```

Comment:
Also, this doesn't let you change the keys without restarting the client. Would doing something like this work?
```javascript
let instance = null;
let instanceKey;
let instanceSecret;

export default function (key, secret) {
    if (!instance || key !== instanceKey || secret !== instanceSecret) {
        instance = new Gfycat({client_id: key, client_secret: secret});
        instanceKey = key;
        instanceSecret = secret;
    }
    return instance;
}
```

Reply:
I updated this to work the way you say, but I'm not sure I needed to.  Switching from the System Console seems to refresh the page.